plugins/vpp/puntplugin/puntplugin.go

```diff
@@ -83,10 +93,10 @@ func (p *PuntPlugin) Init() (err error) {
 		return err
 	}
 
-	// init and register IP punt redirect
-	p.ipRedirectDescriptor = descriptor.NewIPRedirectDescriptor(p.puntHandler, p.Log)
-	ipRedirectDescriptor := adapter.NewIPPuntRedirectDescriptor(p.ipRedirectDescriptor.GetDescriptor())
-	err = p.KVScheduler.RegisterKVDescriptor(ipRedirectDescriptor)
+	// init and register punt exception descriptor
+	p.puntExceptionDescriptor = descriptor.NewPuntExceptionDescriptor(p.puntHandler, p.Log)
```

Comment:
You can you shorten descriptor registration (as for the [ifplugin ](https://github.com/ligato/vpp-agent/blob/dev/plugins/vpp/ifplugin/ifplugin.go#L160) for example).

Reply:
Perhaps later with all other ones.